src/App.tsx

```diff
@@ -1,6 +1,25 @@
+import Mousetrap from 'mousetrap'
 import React from 'react'
 import { BrowserRouter } from 'react-router-dom'
 
+// Enable to view the event atrributes
+// document.addEventListener('keydown', event => {
+//   console.log('==============================')
+//   console.log('==============================')
+//   console.log('==============================')
+//   console.log('Event:', event)
+//   console.log('Keyboard Event object keys:', {
+//     charCode: event.charCode,
+//     code: event.code,
+//     key: event.key,
+//     keyCode: event.keyCode,
+//     metaKey: event.metaKey,
+//   })
+//   console.log('==============================')
+//   console.log('==============================')
+//   console.log('==============================')
+// })
```

Comment:
Uncomment this whole `document.addEventListener` in order to view the event in:
- the console from manual keyboard shortcut entry.
- the test output when running `yarn test` in your terminal.